packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -91,52 +106,35 @@ const MT5AccountCard = ({
                                         <p className='mt5-account-card--paragraph'>{spec_attribute}</p>
                                     </td>
                                     <td className='mt5-account-card__specs-table-data'>
-                                        <p className='mt5-account-card--paragraph'>{specs[spec_attribute]}</p>
+                                        <p className='mt5-account-card--paragraph mt5-account-card--paragraph-right'>
+                                            {specs[spec_attribute]}
+                                        </p>
                                     </td>
                                 </tr>
                             ))}
                         </tbody>
                     </table>
                 </div>
-                {existing_data?.login && is_logged_in && <LoginBadge display_login={existing_data.display_login} />}
+            </div>
+        );
+    };
 
-                {((!existing_data && commission_message) || !is_logged_in) && (
-                    <p className='mt5-account-card__commission mt5-account-card--paragraph'>{commission_message}</p>
-                )}
+    const renderFooter = () => {
+        return (
+            <>
```

Comment:
Same here

Reply:
I have discussed it with team leads, apparently there isn't a particular rule for it but i will change it to `React.Fragment` for better readability 👍